### PR TITLE
Allow user-select in ResultContainer header

### PR DESCRIPTION
**Pull request by pariser**
Often my goal in doing a search in Sourcegraph is to find files where I need to make some kind of changes in my editor. To that end, my lack of ability to copy filenames from headers is super frustrating, so much so that I came here to submit this feature request. I know I'm not the only one of my coworkers who has been frustrated by this.

I hope this isn't a controversial change but understand that there may be further implications of this change that might not be ideal. Please let me know if there are further things to do to help move this along.

Test plan: Trivial change, but I'd be happy to follow instructions to verify this somehow.

### CODE REVIEW

**Comment by christinaforney** [comment]
Hi @pariser - thanks so much for the contribution! This definitely seems useful. I'm curious why we chose to not make it selectable in the first place. 

@slimsag, @felixfbecker - do you know?

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ All notable changes to Sourcegraph are documented in this file.
 
 - A `hardTTL` setting was added to the [Bitbucket Server `authorization` config](https://docs.sourcegraph.com/admin/external_service/bitbucketserver#configuration). This setting specifies a duration after which a user's cached permissions must be updated before any user action is authorized. This contrasts with the already existing `ttl` setting which defines a duration after which a user's cached permissions will get updated in the background, but the previously cached (and now stale) permissions are used to authorize any user action occuring before the update concludes. If your previous `ttl` value is larger than the default of the new `hardTTL` setting (i.e. **3 days**), you must change the `ttl` to be smaller or, `hardTTL` to be larger.
 - Sourcegraph now receives [pings](https://docs.sourcegraph.com/admin/pings) on which extensions are activated from Sourcegraph.com.
+- Allow user select in `ResultContainer` headers (to allow for copying filenames)
```
Comment (felixfbecker) [inline CHANGELOG.md]: Tweak to explain this more from the user perspective (vs technical implementation change)

```suggestion
- Allow text selection in search result headers (to allow for e.g. copying filenames)
```

**Comment by tsenart** [comment]
Dear all,

This is your release captain speaking. 🚂🚂🚂

Branch cut for the **3.9 release is scheduled for tomorrow** at 10:00 CEST.

Is this issue / PR going to make it in time? Please change the milestone accordingly.
When in doubt, reach out!

Thank you

File: CHANGELOG.md (hunk 1 above)
Comment (felixfbecker) [reply CHANGELOG.md]: @pariser I'd be happy to merge this and include in this release if could you apply this suggestion :)

Comment (pariser) [reply CHANGELOG.md]: Thanks @felixfbecker !

**Comment by felixfbecker** [comment]
@pariser would you mind resolving merge conflicts?

**Comment by unknwon** [comment]
@felixfbecker I resolved it.

**Pull request merged**